app/views/schools/cohorts/programme_choice_no_early_career_teachers.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, "Your induction programme" %>
+
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: schools_dashboard_path)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l"><%= @cohort.academic_year %> cohort</span>
+    <h1 class="govuk-heading-xl govuk-!-margin-top-2">You are not expecting any early career teachers this year</h1>
+    <p class="govuk-body-l">
+      If you employ early career teachers later this year, you can <%= govuk_link_to "nominate an induction tutor", choose_location_request_nomination_invite_path %> to set up your programme.
```

Comment:
A new start page to that journey has been added very recently

Reply:
Replaced the links with `resend_email_request_nomination_invite_path`